### ISSUE TITLE
animal bug

### ISSUE THREAD

**Issue by kittysmugglers** [crispytwig/Another-Furniture-Mod#45]
whenever i push my cat onto something the player can sit on my cat sits on chair (sort of like what happens when you push a mob into a boat) and makes it not sit anymore :( not sure if its a bug or not but its really killing my vibe

**Comment by Synthestra** [comment]
Could you provide a video of the issue?

**Comment by kittysmugglers** [comment]
https://github.com/starfish-studios/AnotherFurniture/assets/48418469/35bbc8c5-3d34-40ab-9fff-c39730c27167

**Comment by Synthestra** [comment]
Not really an issue, it correctly sits the cat down when it gets put in the seat and sits then up when ejected. it's only the external right clicking of the entity that changes it.

**Comment by kittysmugglers** [comment]
but the animation on logging off and back on reverts to a standing pose and not the correct sitting one

**Comment by Synthestra** [comment]
Ah i see, i'll have that fixed in the 1.20.1 port. won't be for 1.19

**Comment by kittysmugglers** [comment]
ah damn, is it impossible to fix for 1,19.2?

**Comment by Synthestra** [comment]
it is possible, but we're focusing on 1.20.1 and no longer doing any updates for 1.19.2 unless a big issue arises

**Comment by kittysmugglers** [comment]
i think a bug fix is sufficient (can i motivate with financial compensation)

**Comment by Synthestra** [comment]
yes, 10 million grains of sand harvested from all of the worlds beaches, in equal amount

**Comment by kittysmugglers** [comment]
ok ill send it over to your po box in about 12 days
